proxy/src/http/org.rs

```diff
@@ -255,4 +293,43 @@ mod test {
             })
         );
     }
+
+    #[tokio::test]
+    async fn list() {
+        let registry = Arc::new(RwLock::new(registry::Registry::new(
+            radicle_registry_client::Client::new_emulator(),
+        )));
+        let subscriptions = notification::Subscriptions::default();
+        let api = super::filters(
+            Arc::<RwLock<registry::Registry>>::clone(&registry),
+            subscriptions,
+        );
+
+        // Register the org
+        request()
```

Comment:
Instead of calling the API and therefore depend on the correctness of other filters it would be more straightforward to call methods on `registry` directly to set up the state we need.

Reply:
Esp. as we don't inspect if the request was successful. Calling the methods directly will give use results to panic on.

Reply:
0e5a59c